testonly/encoding.go

```diff
@@ -42,10 +42,7 @@ func MustHexDecode(b string) []byte {
 }
 
 // MustToTimestampProto converts t to a Timestamp protobuf, or panics if this fails.
+// TODO(mhutchinson): inline this.
 func MustToTimestampProto(t time.Time) *tspb.Timestamp {
-	ret, err := ptypes.TimestampProto(t)
-	if err != nil {
-		panic(err)
-	}
-	return ret
+	return timestamppb.New(t)
```

Comment:
I won't say CheckValid here as we don't really want the panic back and it's going to be inlined.

Reply:
In the cold light of a different day, I actually feel like panicking here is reasonable. It's what the docs say anyway, and the other methods in this class do the same thing. It probably does help catch errors sooner than later.